### PR TITLE
instr(kafka): More broker stats

### PR DESCRIPTION
**Pull request by jjbayer**
Follow-up to #3315: Also emit `int_latency` and `outbuf_latency` metrics. 

#skip-changelog

### CODE REVIEW

File: relay-kafka/src/producer/utils.rs [hunk 1]
```diff
@@ -20,7 +20,7 @@ impl ClientContext for Context {
         relay_statsd::metric!(gauge(KafkaGauges::MessageSize) = statistics.msg_size);
         relay_statsd::metric!(gauge(KafkaGauges::MessageSizeMax) = statistics.msg_size_max);
 
-        for broker in statistics.brokers.values() {
+        for (_, broker) in statistics.brokers {
```
Comment (Dav1dde) [inline relay-kafka/src/producer/utils.rs:23]: Why this change?

**Review by Dav1dde — APPROVED** [review]

File: relay-kafka/src/producer/utils.rs (hunk 1 above)
Comment (jjbayer) [reply relay-kafka/src/producer/utils.rs:23]: It was either this or `if let Some(int_latency) = &broker.int_latency {`, and I thought this function might as well consume the object.
